ts/components/conversation/ConversationHeader.tsx

```diff
@@ -245,21 +262,36 @@ const ConversationHeaderTitle = () => {
     text = i18n('members', [count]);
   }
 
-  const textEl =
-    text === '' || isKickedFromGroup ? null : (
-      <span className="module-conversation-header__title-text">{text}</span>
-    );
-
+  const notificationSetting = useSelector(getCurrentNotificationSettingText);
+  const notificationSubtitle = notificationSetting
+    ? window.i18n('notificationSubtitle', notificationSetting)
+    : null;
   const title = profileName || name || phoneNumber;
 
   return (
     <div className="module-conversation-header__title">
       <span className="module-contact-name__profile-name">{title}</span>
-      {textEl}
+      <StyledSubtitleContainer margin={useTheme().common.margins.xs}>
+        {isKickedFromGroup ? null : <ConversationHeaderSubtitle text={text} />}
+        <ConversationHeaderSubtitle text={notificationSubtitle} />
```

Comment:
one of those line should be removed right?

Reply:
Top line is for the conditionally rendered member count subtitle and the other is the notification setting. I can rename the variable for the member count so it's clearer.